daemon/algod/api/algod.oas2.json

```diff
@@ -3108,6 +3108,29 @@
         }
       }
     },
+    "StackValue": {
```

Comment:
ah, this is based on a discussion with Jason, see https://github.com/algorand/go-algorand/pull/5492#issuecomment-1604999517, which is basically 2 things:
1. `TealValue` marks `uint` and `bytes` as both required, but we only need one of these fields
2. `TealValue`'s `bytes` field is just string, so there is a need of `ToTealValue` to b64-ify bytes, while current `StackValue` has format `byte`, such that b64 encoding is handled by generated code.

Reply:
We thought changing `TealValue` might be breaking many things, while it should be fine to introduce a better representation of avm value in endpoint, and deprecate usages of `TealValue` later.

Reply:
I don't think this should be "StackValue". If we're unwilling to use the existing TealValue here, we will surely feel that way again when we expose something else in the trace. So, for example, if we serialize a scratch value, or a global/local state value, we will need the same object, and it will no longer make sense to call it _Stack_ Value.

My preference is to accept that we have a suboptimal definition of TealValue, and use it anyway, so we have consistency rather that one "bad" TealValue and one "good" one. But I don't feel all that strongly.

But I do feel strongly that this isn't a "StackValue".  Maybe "AvmValue"?